magicctapipe/scripts/lst1_magic/config_general.yaml

```diff
@@ -0,0 +1,26 @@
+mc_tel_ids:
```

Comment:
Why having this file twice (here and in resources)? Just leave the one in resources

Reply:
It is needed by the semi-automatic scripts (as the config.yaml is needed by the non-automatic scripts)

Reply:
Still I do not see why it should stay there. In my view, the user should be able to run the semi-automatic pipeline from anywhere, and the configuration file could be in the home directory of the user. The ones in the repository are only examples that the user takes and modifies according to the needs. So `resources` is the place to have the template configuration file. Also, now you updated one with the LST data version, but not the other. Having the same file in two places can create the situation where you update one but not the other :)

Reply:
:facepalm:, I removed it from lst1_magic, so MAYBE I won't make a mess again...